### ISSUE TITLE
import error

### ISSUE THREAD

**Issue by 776488326** [RIAEvangelist/js-message#3]
Hello, the latest update has caused dependency errors in other modules that use your library. Can you please address this issue?
Please provide a solution to this problem so that other modules can continue to work as expected.

eg:
/home/jenkins/agent/workspace/aone-public-compile-job/node_modules/node-ipc/dao/client.js:6
Message = require('js-message'),

**Comment by XingToYang** [comment]
specify a older version of js-message in your root package-lock.json or yarn.lock

**Comment by dserranox** [comment]
Hi! I had this issue when I changed node version (from 12.19.0 to 14.10.01) because after publising new version of js-message I got this error

`error js-message@2.1.0: The engine "node" is incompatible with this module. Expected version ">=13.2.0". Got "12.19.0"
error Found incompatible module.`


I SOLVED this issue adding this in the package.json

`"resolutions": {
    "js-message": "1.0.7"
  }`

**Comment by RIAEvangelist** [comment]
Was the only issue the node version?

**Comment by RIAEvangelist** [comment]
@776488326 was this the same issue for you, the node version? The reason it now specifies node >13.2 ia because the latest versions use ES6 module syntax.

**Comment by dserranox** [comment]
@RIAEvangelist No! sorry if I didn´t explain well. When js-message version was changed I recibed this error:

`error js-message@2.1.0: The engine "node" is incompatible with this module. Expected version ">=13.2.0". Got "12.19.0"
error Found incompatible module.`

For this reason, I changed node version to 14.10.01 and, when I did this, I got @776488326 error. 

 The way to solve this problem was adding this to package.json

`"resolutions": { "js-message": "1.0.7" }`

 Anyway, this solution works changing node version or not
